iOS/MMIWG2S/ViewControllers/ViewController1.swift

```diff
@@ -0,0 +1,163 @@
+    var pc2View:UIView = UIView(frame: CGRect(x:0, y:0, width:0, height:0))
+    var pc3View:UIView = UIView(frame: CGRect(x:0, y:0, width:0, height:0))
+    var pc4View:UIView = UIView(frame: CGRect(x:0, y:0, width:0, height:0))
+    var delegate: NextPageDelegate?
+    
+    override func viewDidLoad() {
+    
+        super.viewDidLoad()
+        let image = UIImage(named: "image1")
+        let imageView = UIImageView(image:image)
+        imageView.frame = view.frame
+        imageView.contentMode = .scaleAspectFill
+        imageView.clipsToBounds = true
+        view.addSubview(imageView)
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            imageView.topAnchor.constraint(equalTo: view.topAnchor),
+            imageView.widthAnchor.constraint(equalTo: view.widthAnchor, multiplier: 1),
+            imageView.heightAnchor.constraint(equalTo: view.heightAnchor),
+            imageView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+            ])
+        
+        
+        let gradient = UIImage(named:"Rectangle1.png")
+        let gradientView = UIImageView(image:gradient)
+        gradientView.frame = view.frame
+        gradientView.contentMode = .scaleAspectFill
+        gradientView.clipsToBounds = true
+        view.addSubview(gradientView)
+        gradientView.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            gradientView.topAnchor.constraint(equalTo: view.topAnchor),
+            gradientView.widthAnchor.constraint(equalTo: view.widthAnchor, multiplier: 1),
+            gradientView.heightAnchor.constraint(equalTo: view.heightAnchor),
+            gradientView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+            ])
+
+        setupUI()
+    }
+    
+    func setupUI() {
+        
+        //Setup UI
+        let labelMMIW = UILabel(frame:CGRect(x:0,y:0,width:50,height:21))
+        let labelMMIWInfo = UILabel(frame:CGRect(x:0,y:0,width:200,height:200))
+        let buttonNext = UIButton(frame:CGRect(x:0,y:0, width:30, height:17))
+         
+        labelMMIW.translatesAutoresizingMaskIntoConstraints = false
+        labelMMIWInfo.translatesAutoresizingMaskIntoConstraints = false
+        buttonNext.translatesAutoresizingMaskIntoConstraints = false
+
+        view.addSubview(labelMMIW)
+        view.addSubview(labelMMIWInfo)
+
+        pc1View.translatesAutoresizingMaskIntoConstraints = false
+        pc2View.translatesAutoresizingMaskIntoConstraints = false
+        pc3View.translatesAutoresizingMaskIntoConstraints = false
+        pc4View.translatesAutoresizingMaskIntoConstraints = false
+    
+        labelMMIW.textAlignment = .center
+        labelMMIW.text = NSLocalizedString("slideviewheader", comment: "Text")
+        labelMMIW.textColor = UIColor.white
+        labelMMIW.font = UIFont(name: "Roboto-Regular", size:36.0 )
```

Comment:
Please move fonts into the `extension UIFont` section of Extensions.swift as static properties.